README.yaml

```diff
@@ -28,9 +28,23 @@ include:
   - "docs/targets.md"
 
 usage: |-
-  1. Run `make init`
-  2. Run `COMPONENTS_BUILD=true/false make run` to start local server
-  3. Run `make build` to generate static site in `public/`
+  1. Download [Docker](https://www.docker.com/).
+  2. Install the [Atom editor](https://atom.io/)(optional).
+
+    * After installation, run `make deps/atom`.
+
+    * Select _Activate the Atom_ &#8594; _Install Shell Commands_ menu item.
```

Comment:
You can use the html entity, but let's just use thee raw utf8 character (→), since that's supported by modern browsers.